node.go

```diff
@@ -109,9 +117,47 @@ func getProtocolParams(cfg *Config) *localstatequery.CurrentProtocolParamsResult
 	if err != nil {
 		return result
 	}
-	// Start our client
-	o.LocalStateQuery().Client.Start()
-	result, err = o.LocalStateQuery().Client.GetCurrentProtocolParams()
+	// Query our client
+	oConn.LocalStateQuery().Client.Start()
```

Comment:
You could drop this entirely now

Reply:
No. I need to upgrade gOuroboros for that and I'm avoiding that for the moment to keep the current version compatible with 1.35.7 and will upgrade that at a later date.